### ISSUE TITLE
sha256 checksum error downloading trinity 2.2.0 dependency

### ISSUE THREAD

**Issue by lparsons** [galaxyproject/tools-iuc#959]
Attempting to install `package_trinity_2_2_0` results in:

```
Given sha256 checksum does not match with the one from the downloaded file (1191c0022bcba15e5b896f6197ac91bc7fb4679e150e9c32ad4181a812ec74f1 != f16b6316fdbd7ab0077644d41a3286a688b0b75cc34686383e10f139f0955ae1).
```

**Comment by nsoranzo** [comment]
That's for https://depot.galaxyproject.org/software/Biobase/Biobase_2.30.0_src_all.tar.gz and indeed the sha256 has changed. @erasche?

**Comment by erasche** [comment]
Still on vacation. But bioarchive tarballs are not reproducible

22. sep. 2016 6.24 p.m. skrev "Nicola Soranzo" <notifications@github.com>:

**Comment by martenson** [comment]
@erasche what do you mean by that?

**Comment by erasche** [comment]
We cannot rebuild the tarballs with the same hashes. Additionally tarballs
we build will differ to those of every other bioc archive service out
there. Again, travelling home from vacation but am guessing this is more
cargo port disaster cleanup. Hashes need to be updated, packages
reuploaded, etc. :/

Den fre. 23. sep. 2016, 22.19 skrev Martin Cech <notifications@github.com>:

**Comment by martenson** [comment]
Seems like we need one hash per url. And dynamic urls without hashes.

**Comment by erasche** [comment]
Ok, no, with @martenson now, not understanding why bioarchive is giving a different result. That's unexpected.

**Comment by lparsons** [comment]
Is this something we can fix by updating `package_trinity_2_2_0`? I'd like to resolve this since my Galaxy instance seems to keep trying to use this instead of falling back to conda.

**Comment by nsoranzo** [comment]
Fixed in #1028.
